test/Microsoft.Framework.PackageManager.FunctionalTests/DnuCommandsTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Framework.CommonTestUtils;
+using Microsoft.Framework.Runtime.DependencyManagement;
+using Xunit;
+
+namespace Microsoft.Framework.PackageManager
+{
+    public class DnuCommandsTests
+    {
+        public static IEnumerable<object[]> RuntimeComponents
+        {
+            get
+            {
+                return TestUtils.GetRuntimeComponentsCombinations();
+            }
```

Comment:
Why not put them on one line to save more lines of code to show up in on your screen?